frontend/src/main/components/Courses/SchoolDropdown.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Form } from "react-bootstrap";
+import {useFormContext} from "react-hook-form";
+
+//This is roughly based on the schedule dropdown from Courses, available
+//at https://github.com/ucsb-cs156/proj-courses/blob/main/frontend/src/main/components/PersonalSchedules/PersonalScheduleDropdown.js

```

Comment:
```suggestion
```

Suggest removing this (which you can do by clicking on the button that says something like `Accept Suggestion` or `Commit Suggestion`.)

I think this might go better in the PR description; otherwise we'd be cluttering up the code with all kinds of similar comments. 

I typically use a comment in the code like this only when it's something *really* unusual that came from an external source (e.g. Stackoverflow, etc.) rather than from within the courses own code bases.